data/application.css

```diff
@@ -244,3 +248,36 @@ row:selected:backdrop.contact-row {
 .desc {
   font-size: 14px;
 }
+
+.attachment-header {
+  background-color: #3b4140;
+  padding: 5px;
+}
+
+.attachment-header .description-box {
+  padding: 10px;
+}
+
+.attachment-preview {
+  background-color: #3b4140;
+  padding: 20px;
+}
+
+.thumbnail .magnifying-glass-overlay {
+  background-image: url('resource:////com/endlessm/Coding/Chatbox/img/normal_zoom.png');
+  background-size: 22px;
+  background-repeat: no-repeat;
+  background-position: 100% 25%;
+  border-top-right-radius: 5px;
+}
+
+.thumbnail:hover .magnifying-glass-overlay {
+  background-image: url('resource:////com/endlessm/Coding/Chatbox/img/over_zoom.png');
+}
+
+.close-button {
+  background-image: url('resource:///com/endlessm/Coding/Chatbox/img/close_button.svg');
```

Comment:
I am afraid this file is not part of the PR? But why use a custom icon for this, instead of the regular one coming from the theme?